cookbook/docs/aws.rst

```diff
@@ -23,7 +23,16 @@ Flyte integrates with various AWS services. The subsections below provide exampl
        :text: Pytorch with Sagemaker
        :classes: btn-block stretched-link
     ^^^^^^^^^^^^
-    Train Pytorch models using Sagamaker, with support for distributed training.
+    Train Pytorch models using Sagemaker, with support for distributed training.
+
+    ---
+
+    .. link-button:: auto/integrations/aws/athena/index
+       :type: ref
+       :text: AWS Athena
+       :classes: btn-block stretched-link
+    ^^^^^^^^^^^^
+    Execute queries using AWS Athena
 
 
 .. toctree::
```

Comment:
add `auto/integrations/aws/athena/index` to the toctree below

Reply:
done, thanks again